src/components/Grid/Grid.tsx

```diff
@@ -66,6 +73,7 @@ const Grid = ({ alert, setAlert, garden, setGarden, bullDoze, setBullDoze, filte
   return (
     <section id='grid'>
       {cells}
+      {popUp && <ConfirmModal setPopUp={setPopUp} fullClear={fullClear} setFullClear={setFullClear} setPartialClear={setPartialClear} reset={reset} setBullDoze={setBullDoze} setFilterGarden={setFilterGarden} setAlert={setAlert} />}
       {modal && <Modal alert={alert} bullDoze={bullDoze} filterGarden={filterGarden} toggleModal={toggleModal} />}
```

Comment:
K i've looked at this for about 30 minutes now and can't determine a cause, but it looks there are a few issues:
1) We lost an alert modal when dragging a Card, and now a Cell, over a `planted` cell
2) If you drag an unplanted plant over a disabled cell (which causes an alert modal), the unplanted plant disappears
3) Dragging a Card, (and now a Cell), over a cell with a non-planted plant doesn't overwrite it anymore - is this intended?

Reply:
all of these bugs have been fixed!